pallets/token_migration/src/lib.rs

```diff
@@ -97,51 +97,51 @@ decl_module! {
         }
 
```

Comment:
What are your thoughts on returning `Pays::No` from `migrate()` so we can remove the `OnlyMigrator` signed extension?

Reply:
Signed extension allows you to make calls when you have no balance at all whereas `Pays::No` will require a balance, however it will not change post the extrinsic